lib/jsanalyze.js

```diff
@@ -160,6 +161,22 @@ exports.analyzeJs = function analyzeJs(contents, opts) {
 	// convert the object of symbol names to an array of symbol names
 	results.symbols = Object.keys(symbols);
 
+	// transpile
+	if (opts.transpile) {
```

Comment:
May want to consider building up the JS object options for babel.transform so that when we transpire and minify we don't do it in two separate calls.

It's entirely possible that it may not matter, though. I'm just guessing it may have a performance impact.

Might want to try a quick test and see...